src/auth/auth.controller.ts

```diff
@@ -38,4 +46,25 @@ export class AuthController {
   async register(@Body() userDto: CreateUserDto) {
     return this.authService.register(userDto);
   }
+
+  @Post("login")
+  @UseGuards(LocalAuthGuard)
+  @ApiOperation({
+    description:
+      "email or account as login account.  \n" +
```

Comment:
The description about "account" parameter should be moved into the property description instead of here as a API operation description.

Reply:
Revised.
[refactor: revised grammar questions about words](https://github.com/Cophr/backend/pull/10/commits/efd53265a8d42a602c74c96c85cd0166208335e9)